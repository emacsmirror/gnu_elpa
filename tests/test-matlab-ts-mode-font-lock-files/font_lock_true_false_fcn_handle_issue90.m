% -*- matlab-ts -*-

% see https://github.com/acristoffers/tree-sitter-matlab/issues/90

if someCondition
    h = @true;
else
    h = @false;
end

h(5)
