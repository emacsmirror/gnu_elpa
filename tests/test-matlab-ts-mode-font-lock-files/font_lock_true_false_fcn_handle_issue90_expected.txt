c ccc ccccccccc ccc

c ccc cccccccccccccccccccccccccccccccccccccccccccccccccccccccccccc

kk ddddddddddddd
    v o oBBBBD
kkkk
    v o oBBBBBD
kkk

dbnb
